[MRG+1] sklearn: neighbours: dist_metrics.pyx: Documentation Fix of issue #10396

pinakinathc opened the pull request:
DistanceMetric formatting fix for haversine distance_function

Signed-off-by: Pinaki Nath Chowdhury <pinakinathc@gmail.com>

<!--
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#pull-request-checklist
-->

#### Reference Issues/PRs
<!--
Example: Fixes #1234. See also #3456.
Please use keywords (e.g., Fixes) to create link to the issues or pull requests
you resolved, so that they will automatically be closed when your pull request
is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests
-->
Fixes #10396 

#### What does this implement/fix? Explain your changes.


#### Any other comments?


<!--
Please be aware that we are a loose team of volunteers so patience is
necessary; assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.
For more information, see our FAQ on this topic:
http://scikit-learn.org/dev/faq.html#why-is-my-pull-request-not-getting-any-attention.

Thanks for contributing!
-->

pinakinathc committed:
sklearn: neighbours: dist_metrics.pyx: Documentation Fix of issue #10396

DistanceMetric formatting fix for haversine distance_function

Signed-off-by: Pinaki Nath Chowdhury <pinakinathc@gmail.com>

qinhanmin2014 commented:
Seems not working. See [the rendered doc](https://16296-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.neighbors.DistanceMetric.html#sklearn.neighbors.DistanceMetric).

pinakinathc committed:
sklearn: neighbors: dist_metrics.pyx: Fix of documentation error of issue #10396

Documentation BUG in DistanceMetric formatting for haversine distance_function

Signed-off-by: Pinaki Nath Chowdhury <pinakinathc@gmail.com>

qinhanmin2014 commented:
@pinakinathc It would be better to post your comment on GitHub instead of sending me an email :)
Circle CI will automatically build the doc. See [contributing guide](http://scikit-learn.org/dev/developers/contributing.html#documentation).

pinakinathc commented:
@qinhanmin2014 can you help me understand why did the Travis CI build failed?

pinakinathc committed:
sklearn: neighbors: Documentation bug fix issue #10396

Fixing documentation bug in Haversine Distance at line 147

Signed-off-by: Pinaki Nath Chowdhury <pinakinathc@gmail.com>

qinhanmin2014 reviewed (approved):
LGTM. I'm wondering if we can put the function in two lines while remaining the format.

jnothman commented:
Thanks @pinakinathc

pinakinathc commented:
Thanks a lot, @jnothman  @qinhanmin2014 for helping me.
